.github/workflows/deploy-release.yml

```diff
@@ -0,0 +1,55 @@
+name: Deploy Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Existing tag to be released, e.g. v1.2.3, v2.2.2, v3.0.0'
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Deploy release
+    steps:
+      - name: Determine tag to be released
+        run: |
+          TAG=${GITHUB_REF:10}
+          if [ -n "${{ github.event.inputs.tag }}" ]; then
```

Comment:
Any need for `if` is unput is required?

Reply:
The workflow also triggered with a tag push, the input will be null in that case

Reply:
Got it. thanks